mne/preprocessing/maxwell.py

```diff
@@ -502,6 +579,7 @@ def _sss_basis(origin, coils, int_order, ext_order, mag_scale=100.,
                          for coil in coils])] = mag_scale
 
     # Compute internal/external basis vectors (exclude degree 0; L/RHS Eq. 5)
+    mu_0 = 4e-7 * np.pi  # magnetic permeability
```

Comment:
can you share the evoked plots for:

- original raw data
- maxfilter elekta output
- mne-pyton maxwell_filter output

?